### ISSUE TITLE
Sympa Accessibility Issue - 19 - Moderate option is not keyboard accessible

### ISSUE THREAD

**Issue by rm1-uiuc** [sympa-community/sympa#1750]
The University of Illinois at Urbana-Champaign, Technology Services Accessibility Testing team conducted an Accessibility Review of Sympa on September 25, 2023 and found 26 issues that need to be fixed to make the interface accessible for visual impaired users.

Version
-------
6.2.72

Installation method
-------------------
Source Package

Expected behavior
-----------------
“Moderate” option should be keyboard accessible, the submenu must be viewed when the screen size is reduced.

Actual behavior
---------------
“Moderate” option is not keyboard accessible, the submenu for that cannot be viewed when the screen size is reduced.

Steps to reproduce
------------------
Sympa Accessibility Issue -19 was determined by manual functional testing performed by the user navigating with keyboard only or using assistive technology such as screen reader. See attached review for more info on specific location in interface.

“Moderate” option is not keyboard accessible, the submenu for that cannot be viewed when the screen size is reduced.

[2.1.1 Keyboard](https://www.w3.org/WAI/WCAG21/Understanding/keyboard.html)

![image](https://github.com/sympa-community/sympa/assets/144282683/8ec3ad6e-04a9-4b98-86cd-423cdc0bb960)
Fig 22 Keyboard accessibility issue.

Additional information
----------------------
Attached is the Accessibility Review conducted on Sept 25,2023
[Tech Services Review - Accessibility Evaluation Report for Sympa.pdf](https://github.com/sympa-community/sympa/files/13505318/Tech.Services.Review.-.Accessibility.Evaluation.Report.for.Sympa.pdf)

**Comment by ikedas** [comment]
Hi @rm1-uiuc , thank you so much for your review!

> “Moderate” option is not keyboard accessible, the submenu for that cannot be viewed when the screen size is reduced.

I don't think so.  When the item has submenu, i.e. with a triangle on the right side, appropriate keyboard action such as typing Enter key expands submenu.  Thus I think the submenu can be keyboard accessible.

**Comment by ldidry** [comment]
> Thus I think the submenu can be keyboard accessible.

Indeed. It’s the same mechanism as for the sub-menus in top bar.

**Comment by ikedas** [comment]
If no objection, this issue will be closed with [invalid] (not a bug) label.

See also #1749 .
